src/Mod/TechDraw/Gui/ViewProviderPage.cpp

```diff
@@ -231,7 +231,7 @@ void ViewProviderPage::setupContextMenu(QMenu* menu, QObject* receiver, const ch
     Gui::ViewProviderDocumentObject::setupContextMenu(menu, receiver, member);
     QAction* act = menu->addAction(QObject::tr("Show drawing"), receiver, member);
     act->setData(QVariant((int)ShowDrawing));
-    QAction* act2 = menu->addAction(QObject::tr("Toggle Keep Updated"), receiver, member);
+    QAction* act2 = menu->addAction(QObject::tr("Toggle keep updated"), receiver, member);

```

Comment:
```suggestion
    QAction* act2 = menu->addAction(QObject::tr("Toggle Keep Updated"), receiver, member);

```

Command in (context)menu (addAction)